### PR TITLE
Stop all apiserver at the same time when restarting.

### PR DESCRIPTION
**Pull request by kfyharukz**
This PR is to prevent that different version `kube-apiserver`s exist at the same time by stop `kube-apiserver` simultaneously.
ref: https://github.com/cybozu-go/neco/issues/880

### CODE REVIEW

File: op/common/container.go [hunk 1]
```diff
@@ -109,42 +109,52 @@ func (c runContainerCommand) Command() cke.Command {
...
-		return nil
-	}
-	// Inspect returns ServiceStatus for the named container.
-	statuses, err := ce.Inspect([]string{c.name})
-	if err != nil {
-		return err
-	}
-	if st, ok := statuses[c.name]; ok && st.Running {
-		err = ce.Stop(c.name)
+	for _, n := range c.nodes {
```
Comment (ymmt2005) [inline op/common/container.go:127]: Make this parallel, just like `Run`.

**Comment by ymmt2005** [comment]
@kfyharukz @dulltz 
You need to modify these lines too:
https://github.com/cybozu-go/cke/blob/master/op/k8s/apiserver_restart.go#L53-L55

**Comment by ymmt2005** [comment]
@kfyharukz @dulltz 
One more point.  It is better to exchange `APIServerBootOp` and `APIServerRestartOp`
in [server/stragegy.go](https://github.com/cybozu-go/cke/blob/master/server/strategy.go#L102-L107) because otherwise there could be mixed API server versions
if CKE is killed during `APIServerRestartOp` and only part of API servers are stopped.

File: op/common/container.go (hunk 1 above)
Comment (dulltz) [reply op/common/container.go:127]: OK!

**Pull request merged**